transmit-lib/src/cmr/transmit/kms.clj

```diff
@@ -256,11 +256,12 @@
   ;; process is driven by the config value `kms_scheme_overrides`
   ;; which contains a string of JSON such as:
   ;;     "{\"platforms\":\"static\"}"
+  {:pre (config/context->app-connection context :kms)}
```

Comment:
adding this test to help you if you construct a context by hand and forget to have KMS included.